3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -27,19 +27,19 @@ exercise file.
 const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
-  // TODO Refactor this function
-  const dice = [1, 2, 3, 4, 5];
-  return rollDie(1);
+    const dice = [1, 2, 3, 4, 5];
+    const promises = dice.map((die) => rollDie(die))
+    return Promise.all(promises)
```

Comment:
code style: assigning the output of this dice.map to promises isn't giving you any real benefit, if you don't reuse a variable you can just inline it

```suggestion
    return Promise.all(dice.map(rollDie))
```

Reply:
Thank you. More readable than mine.